src/main/scala/com/atomist/rug/function/github/CreateTagFunction.scala

```diff
@@ -35,15 +31,14 @@ class CreateTagFunction
     logger.warn(s"Invoking createTag with tag '$tag', message '$message', sha '$sha', owner '$owner', repo '$repo', apiUrl '$apiUrl' and token '${safeToken(token)}'")
 
     val apiUrlSlash = if (apiUrl.endsWith("/")) apiUrl else s"$apiUrl/"
```

Comment:
Maybe a good idea to move this logic into the `gitHubServices(token, url)` function?